src/screens/Dashboard.js

```diff
@@ -0,0 +1,198 @@
+/* eslint-disable react/destructuring-assignment */
+import React, { Component } from "react";
+import { bindActionCreators } from "redux";
+import { connect } from "react-redux";
+import {
+  withStyles, AppBar, Toolbar, Typography, Button, Avatar,
+  TextField, InputAdornment,
+} from "@material-ui/core";
+import { Refresh, Search } from "@material-ui/icons";
+import ReactTable from "react-table";
+import matchSorter from "match-sorter";
+import inflection from "inflection";
+
+import { retrieveUserList } from "store/ducks/List";
+import { bigStyles, StyledTable, TextButton } from "screens/DashboardStyles";
+import ModalCard from "components/ModalCard";
+
+const defaultState = {
+  selectedUser: [
+    "",
+    {
+      birth_date: "",
+      phone_number: "",
+    },
+  ],
+  openModal: false,
+};
+
+class Dashboard extends Component {
+  constructor(props) {
+    super(props);
+    this.reactTable = React.createRef();
+  }
+
+  state = defaultState;
+
+  async componentDidMount() {
+    this.refreshList();
+  }
+
+  shouldComponentUpdate(nextProps, prevState) {
+    if (prevState.openModal !== this.state.openModal) {
```

Comment:
this is probably wrong, so only openModal change then we rerender ?
this is naive checking.